src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -8,6 +8,9 @@ namespace Microsoft.Maui
 {
 	public static class ViewExtensions
 	{
+		const int DefaultAutomationTagId = -1;
```

Comment:
Is this the best place for this? Where is the final place "planned" Is this a context thing or a app builder thing? Maybe it can be here but set via another way?